master/buildbot/db/model.py

```diff
@@ -104,6 +104,7 @@ class Model(base.DBConnectorComponent):
     # claim is made by the master referenced by masterid.
     buildrequest_claims = sautils.Table(
         'buildrequest_claims', metadata,
+        sa.Column('dummy', sa.Integer, primary_key=True, autoincrement=True),
         sa.Column('brid', sa.Integer, sa.ForeignKey('buildrequests.id'),
```

Comment:
Indeed, it is weird, but it is quite important.

The reason why there is this table is to implement the buildrequest claiming in an atomic way. 

In a multimaster environment, a buildrequest is created in the buildrequest table, and then a message will be sent to all masters to inform them about it.

Each master capable of building this request will then try to insert an entry in the claim table, the rdbms guarantees that only one master will manage to insert that entry. The other master will get that query cancelled, and then it will know that it shouldn't build this request as another master did take it.